pontoon/contributors/static/css/profile.css

```diff
@@ -35,6 +35,27 @@ h2 {
   border-color: var(--dark-grey-1);
 }
 
+.avatar .status {
+  box-sizing: border-box;
+  display: flex;
+  width: 100%;
+  height: 24px;
+  align-items: center;
+  justify-content: center;
+  gap: 4px;
+  position: absolute;
+  bottom: 3px;
+  border-bottom-left-radius: 6px;
+  border-bottom-right-radius: 6px;
+  background: var(--button-background-2);
```

Comment:
Please set the color to `var(--dark-grey-1)`, so that it matches the avatar border.

The mismatch is particularly noticeable in the light theme.

Reply:
Ahh great catch, the suggestion looks much nicer 👍